tool/report_static.py

```diff
@@ -139,31 +139,31 @@ def write_summary(df, project_name, release_version, package_manager, filename,
     warning_counts = {}
     if enabled_checks.get("source_code"):
         warning_counts["no_source_code"] = (
-            f":heavy_exclamation_mark: Packages with no Source Code URL(⚠️⚠️⚠️) {(df['github_url'] == 'No_repo_info_found').sum()}"
+            f":heavy_exclamation_mark: Packages with no source code URL (⚠️⚠️⚠️) {(df['github_url'] == 'No_repo_info_found').sum()}"
         )
         warning_counts["github_404"] = (
-            f":no_entry: Packages with Github URLs that are 404(⚠️⚠️⚠️) {(df['github_exists'] == False).sum()}"
+            f":no_entry: Packages with repo URL that is 404 (⚠️⚠️⚠️) {(df['github_exists'] == False).sum()}"
         )
 
     if enabled_checks.get("release_tags"):
         warning_counts["release_tag_not_found"] = (
-            f":wrench: Packages with accessible source code repos but inaccessible GitHub tags(⚠️⚠️⚠️) {(release_tag_not_found_df.shape[0])}"
```

Comment:
Regarding this line, I had previously added the preamble of "accessible source code repos" to make it clear that we are only looking at repos we could effectively find, but I can see how that is probably redundant